it-tests/BasicFlow.test.ts

```diff
@@ -33,7 +33,7 @@ describe('Kaoto basic development flow', function () {
 			} catch {
 				return false;
 			}
-		}, 50000); // if it is the first load, it can take a lot of time as it downloads the docker image
+		}, 100000); // if it is the first load, it can take a lot of time as it downloads the docker image
```

Comment:
On Jenkins Ci, it is even slower than on Github Workflow, so needed to increase the timeout.